packages/frontend/src/components/send/components/InputAccountId.js

```diff
@@ -190,12 +201,14 @@ class InputAccountId extends Component {
             accountId,
             onFocus,
             onBlur,
-            autoFocus
+            autoFocus,
```

Comment:
Remove risk-scoring specific logic from there

I guess we can just pass 'error' prop to highlight the input red?

Reply:
fixed